spec/rails_helper.rb

```diff
@@ -3,11 +3,41 @@
 require File.expand_path('../../config/environment', __FILE__)
 # Prevent database truncation if the environment is production
 abort("The Rails environment is running in production mode!") if Rails.env.production?
+
+require "codeclimate-test-reporter"
+CodeClimate::TestReporter.start
+require 'simplecov'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.